pwiz_tools/Skyline/TestFunctional/EADZIonsTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Globalization;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using pwiz.Common.SystemUtil;
+using pwiz.Skyline.Controls.SeqNode;
+using pwiz.Skyline.Model;
+using pwiz.Skyline.Properties;
+using pwiz.Skyline.SettingsUI;
+using pwiz.Skyline.Util;
+using pwiz.SkylineTestUtil;
+
+namespace pwiz.SkylineTestFunctional
+{
+    /// <summary>
+    /// Testing z+1 and z+2 ions feature.
+    /// </summary>
+    [TestClass]
+    public class EADZIonsTest : AbstractFunctionalTestEx
+    {
+        [TestMethod]
+        public void TestEADIons()

```

Comment:
What does "EAD" stand for?
We try to avoid three or more capital letters in a row like this, so this function would normally be called "TestEadIons".

Reply:
EAD stands for Electron Assisted Dissociation. It's a standard acronym, hence the capitalization.
To enter z+1 in Transition Settings you use z. or z* aliases, they are mentioned in the tooltip for that textbox.